bin/tag-release.sh

```diff
@@ -1,6 +1,23 @@
 #!/bin/bash
+set -euo pipefail
 
 moz_git_remote="${MOZ_GIT_REMOTE:-origin}"
+do_push=false
+
+# parse cli args
+while [[ $# -ge 1 ]]; do
+  key="$1"
+  case $key in
+    -P|--push)
```

Comment:
I did intentionally use a capital here. It's a mode that affects things in the remote repo, so I wanted to make it seem a bit more intentional, but that's probably overkill...